dali/base/dacsds.cpp

```diff
@@ -52,19 +52,19 @@ static CriticalSection SDScrit;
 
 ////////////////////
 
-class MonitoredChildMap : public ChildMap
+class MonitoredChildMap : public ChildMapAtom
```

Comment:
naming: It matches the existing convention (NC), but it is hard to read.  In english it is normal to specialise a class name by prefixing the class name with the specialism.  i.e. AtomChildMap.  Otherwise it reads as a specialised type of Atom.